CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [0.0.17] - 2023-11-29
 
+## Added
```

Comment:
This section has to move to the [UNRELEASED] part of the `CHANGELOG.md` and be a 3rd level title.